packages/react-components/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -44,7 +44,7 @@ export const usePortalMountNode = (options: UsePortalMountNodeOptions): HTMLElem
     return newElement;
   }, [targetDocument, options.disabled]);
 
-  useIsomorphicLayoutEffect(() => {
+  React.useMemo(() => {
```

Comment:
```suggestion
  // Heads up!
  // ----TBD----
  React.useMemo(() => {
```

Please add some description for that, so it will not be accidentally removed